src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -1035,6 +1036,10 @@ else if (resultSet != null) {
                 SQLServerException.makeFromDriverError(connection, this, form.format(msgArgs), null, true);
             }
         }
+        finally {
+            if (null != stmt)
+                stmt.close();
```

Comment:
this statement creates resultset which will be returned, closing this statement also close the returned resultset, which is not what we want I guess